src/04_printing.py

```diff
@@ -12,6 +12,12 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print("x is %2d, y is %1.5f, z is %s" %(x,y,z))
```

Comment:
for the formatting on a float, the number you are writing is meant to give the specific number of decimal points to go.
since the problem only wanted two points of precision a better way to format y would have been %.2f

Reply:
ahh I was wondering about this!